### ISSUE TITLE
Create factories and tests for the events app in the backend

### ISSUE THREAD

**Issue by to-sta** [activist-org/activist#509]
### Terms

- [X] I have searched [open and closed feature requests](https://github.com/activist-org/activist/issues?q=is%3Aissue+label%3Afeature)
- [X] I agree to follow activist's [Code of Conduct](https://github.com/activist-org/activist/blob/main/.github/CODE_OF_CONDUCT.md)

### Description

The task is to create test data (factories) with [factory_boy](https://factoryboy.readthedocs.io/en/stable/) for the models in the events app. 

```bash
Name                            Stmts   Miss  Cover
---------------------------------------------------
...
events\models.py                   74      9    88%
...
```
Have a look at the `content/factory.py` and `content/tests.py` to get started.

You can run `pytest` inside backend directory and a report terminal and html report will be generated. The html report shows you excatly what needs to be tested. The html report will saved in the folder "htmlcov".

### Contribution

Happy to help someone with this task.

**Comment by iagocarmona** [comment]
Hello, I would like to work on this! Can assign it to me ?

**Comment by andrewtavis** [comment]
Thanks for offering to help, @iagocarmona! Let us know if you need any assistance along the way 😊

**Comment by andrewtavis** [comment]
Closed via #543 🚀 Thank you, @iagocarmona! 🙏
